client/src/components/ActivityShow.js

```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect } from "react";
 import ReviewsShowPage from "./ReviewsShowPage";
 
 const ActivityShow = (props) => {
+  const [reviewList, setReviewList] = useState([]);
```

Comment:
look carefully - reviews is already a piece of state for `activity` - this new state should be removed